### PR TITLE
Default temperature corrections on the any color profiles

### PR DESCRIPTION
**Pull request by pkuiper-ultimaker**
PP-350

# Description

The default print temperature have been normalized to the AA0.4, 0.15mm print modes. The temperature delta's for each print modes thus have changed. The new any color material print profiles were not yet adjusted for these changes, this PR changes that, 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Printer definition file(s)
- [ ] Translations

# How Has This Been Tested?

Check temperatures PLA, T-PLA, PETG and ABS. The Generic and Ultimaker values are the same for all print modes.

CURA-11060

### CODE REVIEW

**Review by Frederic98 — APPROVED** [review]

File: resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.15mm.inst.cfg [hunk 1]
```diff
@@ -37,9 +37,9 @@ jerk_roofing = =jerk_wall_0
 jerk_topbottom = =jerk_wall
 jerk_wall = =jerk_infill
 jerk_wall_0 = =max(30, speed_wall_0/2)
-machine_nozzle_cool_down_speed = 1,3
-machine_nozzle_heat_up_speed = 1,9
-material_extrusion_cool_down_speed = 0,7
+machine_nozzle_cool_down_speed = 1.3
+machine_nozzle_heat_up_speed = 1.9
+material_extrusion_cool_down_speed = 0.7
 material_max_flowrate = 14
```
Comment (saumyaj3) [inline resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.15mm.inst.cfg:43]: material_print_temperature = =default_material_print_temperature +10

File: resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.2mm.inst.cfg [hunk 2]
```diff
@@ -37,11 +37,10 @@ jerk_roofing = =jerk_wall_0
...
 jerk_wall = =jerk_infill
 jerk_wall_0 = =max(30, speed_wall_0/2)
-machine_nozzle_cool_down_speed = 1,3
-machine_nozzle_heat_up_speed = 1,9
-material_extrusion_cool_down_speed = 0,7
+machine_nozzle_cool_down_speed = 1.3
+machine_nozzle_heat_up_speed = 1.9
+material_extrusion_cool_down_speed = 0.7
 material_max_flowrate = 14
-material_print_temperature = =default_material_print_temperature + 5
```
Comment (saumyaj3) [inline resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.2mm.inst.cfg:44]: material_print_temperature = =default_material_print_temperature +15

File: resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.15mm.inst.cfg (hunk 1 above)
Comment (Frederic98) [reply resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.15mm.inst.cfg:43]: Temperature should be the same as [um_s3_aa0.4_tough-pla_0.15mm](https://github.com/Ultimaker/Cura/blob/main/resources/quality/ultimaker_s3/um_s3_aa0.4_tough-pla_0.15mm.inst.cfg)

File: resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.2mm.inst.cfg (hunk 2 above)
Comment (Frederic98) [reply resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.2mm.inst.cfg:44]: Temperature should be the same as [um_s3_aa0.4_tough-pla_0.2](https://github.com/Ultimaker/Cura/blob/main/resources/quality/ultimaker_s3/um_s3_aa0.4_tough-pla_0.2mm.inst.cfg), so also no offset

File: resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.3mm.inst.cfg [hunk 3]
```diff
@@ -38,11 +38,11 @@ jerk_roofing = =jerk_wall_0
...
 jerk_wall_0 = =max(30, speed_wall_0/2)
-machine_nozzle_cool_down_speed = 1,3
-machine_nozzle_heat_up_speed = 1,9
-material_extrusion_cool_down_speed = 0,7
+machine_nozzle_cool_down_speed = 1.3
+machine_nozzle_heat_up_speed = 1.9
+material_extrusion_cool_down_speed = 0.7
 material_max_flowrate = 14
-material_print_temperature = =default_material_print_temperature + 10
+material_print_temperature = =default_material_print_temperature + 5
```
Comment (saumyaj3) [inline resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.3mm.inst.cfg:45]: here temperature of 0.3 mm is not increased.

**Review by saumyaj3 — APPROVED** [review]

**Pull request merged**